openVfsFuse/src/openvfsfuse.cpp

```diff
@@ -58,19 +58,59 @@ using json = nlohmann::json;
 using namespace std;
 
 namespace {
-std::string mountPoint;
 
-std::string getAbsolutePath(const char *path)
+class VFSFuseContext
 {
-    return mountPoint + path;
-}
+public:
+    VFSFuseContext(const std::filesystem::path &mountPoint)
+        : _mountPoint(mountPoint)
+        , _rootHandle(open(_mountPoint.c_str(), 0))
```

Comment:
There needs to be some handling for the fail of this open. It returns -1 in that case.

Not sure, why and how it will go wrong, but it will.